src/conversions.jl

```diff
@@ -1643,7 +1643,10 @@ struct FastPixel end
 Vector of anything that is accepted as a single marker will give each point it's own marker.
 Note that it needs to be a uniform vector with the same element type!
 """
-to_spritemarker(marker::AbstractVector) = map(to_spritemarker, marker)
+function to_spritemarker(marker::AbstractVector{T}) where T
+    spritemarker = map(to_spritemarker, marker)
+    convert(Vector{Union{T,eltype(spritemarker)}}, spritemarker)  # retain original type T
```

Comment:
I think it would be better to restrict it to all possible converted marker types...
The fact that I don't know it out of the top of my head already shows, that this is under documented, and adding better types here would at least help a little with this :) 
I can take a look!

Reply:
Yes, as is, this PR breaks a reference test and I must admit I have no idea left on top of my head now. At least, we now know where the issue lies :)